src/git/log.js

```diff
@@ -11,6 +11,7 @@ function log(repoPath, done) {
     cwd: repoPath
   }, function(error, stdout, stderr) {
     if (error) {
+      throw new Error(error);
```

Comment:
I agree it already is an instance of Error. I'll change that.

The reason I decided to throw here is that this is actually fatal. If we can't do git log, we can't check if the commit worked and the process needs to throw so that our tests can return the correct error code and output an actual error to stderr. It is only used in test and was silently failing in the previous configuration.